packages/lab/src/__tests__/__e2e__/progress/CircularProgress.cy.tsx

```diff
@@ -1,23 +1,22 @@
 import { CircularProgress } from "../../../progress";
 
 describe("GIVEN a CircularProgress", () => {
-  beforeEach(() => {
-    cy.mount(<CircularProgress value={50} />);
-  });
-
   it("SHOULD render progress circle", () => {
```

Comment:
This isn't a great test. It's already covered by the QA story.

Reply:
would you say we just test the min-max one that is the only one that would alter how the calculation is done?

Reply:
That should be fine.